generators/heroku/generator.mjs

```diff
@@ -710,7 +710,48 @@ export default class HerokuGenerator extends BaseGenerator {
 
             this.log.log(chalk.bold('\nConfiguring Heroku'));
             await execCmd(`heroku config:set ${configVars}--app ${this.herokuAppName}`);
-            await execCmd(`heroku buildpacks:add ${buildpack} --app ${this.herokuAppName}`);
+            const { stdout: data } = await execCmd(`heroku buildpacks:add ${buildpack} --app ${this.herokuAppName}`);
+            if (data) {
+              this.logger.info(data);
+              // remote:  !     The following add-ons were automatically provisioned: . These add-ons may incur additional cost,
+              // which is prorated to the second. Run `heroku addons` for more info.
+              if (data.includes('Run `heroku addons` for more info.')) {
+                await execCmd('heroku addons');
+              }
+
+              this.log('');
+              const prompts = [
+                {
+                  type: 'list',
+                  name: 'userDeployDecision',
+                  message: 'Continue to deploy?',
+                  choices: [
+                    {
+                      value: 'Yes',
+                      name: 'Yes, I confirm',
+                    },
+                    {
+                      value: 'No',
+                      name: 'No, abort (Recommended)',
+                    },
+                  ],
+                  default: 0,
```

Comment:
If abort is the recommended option why using '0' aka yes as default option? Besides this it looks fine to me, but didn't do proper testing yet (as my local development setup is broken for whatever reason).